From 064773ba2b8ac3e9a48c17f0ff0b00ded5a13796 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Mon, 20 Feb 2012 13:05:20 +0800
Subject: [PATCH 1058/2463] ENGR00174829-2 power: max8903_charger: fix build
 warrning.

Fix below warrning message:
drivers/power/max8903_charger.c:218:
warning: 'gpio' may be used uninitialized in this function

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 drivers/power/max8903_charger.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/power/max8903_charger.c b/drivers/power/max8903_charger.c
index 19ca5fb..89bb5d1 100644
--- a/drivers/power/max8903_charger.c
+++ b/drivers/power/max8903_charger.c
@@ -185,7 +185,7 @@ static __devinit int max8903_probe(struct platform_device *pdev)
 	struct device *dev = &pdev->dev;
 	struct max8903_pdata *pdata = pdev->dev.platform_data;
 	int ret = 0;
-	int gpio;
+	int gpio = 0;
 	int ta_in = 0;
 	int usb_in = 0;
 
-- 
1.7.7.4

